api/v1alpha1/datamoverbackup_types.go

```diff
@@ -32,8 +32,31 @@ type DataMoverBackupStatus struct {
 	// Include references to the volsync CRs and their state as they are
 	// running
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// Includes source PVC name and size
+	SourcePVCData PVCData `json:"sourcePVCData,omitempty"`
```

Comment:
Question, do we need this? With the CSI plugin integration won't this be irrelevant?

Reply:
We need the temporary PVC details that we create. VolSync restore process would need that info. Technically we can derive the PVC name from the original snapshot name but this would make the process much more simpler.